include/matx/transforms/fft.h

```diff
@@ -956,6 +971,15 @@ __MATX_INLINE__ void ifft(OutputTensor o, const InputTensor i,
   // adjusts size of tensor based on fft_size
   auto in = detail::GetFFTInputView(out, in_t, fft_size, stream);
 
+  // Normalize input if necessary
+  using input_type = decltype(in_t);
+  using s_type = typename detail::value_promote_t<typename input_type::scalar_type>;
+  if (norm == ORTHO) {
+    (in *= std::sqrt(static_cast<s_type>(fft_size))).run(stream);
```

Comment:
The comment says 1/sqrt(N), but this seems to be sqrt(N). which is right?

Reply:
Let me correct the comment. Test helped verify.